src/components/Vqb.vue

```diff
@@ -1,36 +1,40 @@
 <template>
   <div class="visual-query-builder">
     <ResizablePanels>
-      <FormRenameStep
+      <component
         v-if="isEditingStep"
+        :is="formToInstantiate"
         :initialValue="initialValue"
         :isStepCreation="isStepCreation"
         slot="left-panel"
         @cancel="toggleStepEdition"
         @formSaved="saveStep"
-      />
+      ></component>
       <Pipeline v-else slot="left-panel" @editStep="openStepForm"/>
       <DataViewer slot="right-panel" @stepCreated="openStepForm"/>
     </ResizablePanels>
   </div>
 </template>
 
 <script lang="ts">
-import Vue from 'vue';
+import Vue, { VueConstructor } from 'vue';
 import { Component } from 'vue-property-decorator';
 import { Getter, Mutation, State } from 'vuex-class';
 import { VQBState } from '@/store/state';
 import { Pipeline, PipelineStep } from '@/lib/steps';
 import DataViewer from '@/components/DataViewer.vue';
+import FormFillnaStep from '@/components/FormFillnaStep.vue';
```

Comment:
We should have an index that export all the Forms Component here to be more concise into our imports